aip/0194.md

```diff
@@ -25,6 +26,7 @@ Most requests can either be categorized as **retryable**, **transactional**, or
 unintended state changes. Most CRUD RPCs which take a unique identifier are
 considered retryable (see more below in "Idempotency and pragmaticism").
```

Comment:
Should this me `pragmatism`? I found only an [obscure branch of philosophy](https://en.wikipedia.org/wiki/Pragmaticism) for `pragmaticism`.